Bottom: 493b8215f47e1309006bf1172423d76fb793a18e
Top:    1d74f841fecaeda5a1f6499c71b96c5273dac8e5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-16 13:45:32 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
